docs/compiling.rst

```diff
@@ -616,6 +684,36 @@ Building with Bazel
 You can build with the Bazel build system using the `pybind11_bazel
 <https://github.com/pybind/pybind11_bazel>`_ repository.
 
+Building with Meson
+===================
+
+You can use Meson, which has support for ``pybind11`` as a dependency (internally
+relying on our ``pkg-config`` support):
+
+.. code-block:: meson
```

Comment:
This `code-block` is identical to one above. Could this `Building with Meson` section be omitted? Or the `code-block` above be referenced (e.g. simply "see above") instead of copied?

Reply:
It could, the CMake ones have separate blocks, but those are showing something different. This doesn't currently show anything different, but I was matching CMake. I could refer back, instead.

Reply:
Linked to the previous block.